CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. For future plans, see our [Roadmap](https://github.com/precice/precice/wiki/Roadmap).
 
+## 2.0.2
+
+- Fixed a critical bug in the testing framework.
+- Fixed a critical bug in the partitioning with two-level initialization enabled.
```

Comment:
```suggestion
- Fixed a critical bug in the partitioning for geometric filtering on master.
```
This also affects partitioning when two-level init is off.

Reply:
We should probably also mention that this is a setting in the use-mesh tag and that this bug does not affect the default.

```suggestion
- Fixed a critical bug in the partitioning for geometric filter set to `on-master` in `<use-mesh>` tags. The default configuration is `on-slave`.
```

Reply:
I agree